lightning/src/ln/channelmanager.rs

```diff
@@ -2945,6 +2953,13 @@ where
 						// Unfunded channel has no update
 						(None, chan_phase.context().get_counterparty_node_id())
 					},
+					// TODO(dual_funding): Combine this match arm with above.
+					#[cfg(dual_funding)]
+					ChannelPhase::UnfundedOutboundV2(_) | ChannelPhase::UnfundedInboundV2(_) => {
+						self.finish_close_channel(chan_phase.context_mut().force_shutdown(false, ClosureReason::HolderForceClosed));
```

Comment:
Why hardcoded reason is used here (`ClosureReason::HolderForceClosed`), and not the computed `closure_reason`, like in the arms above?

Reply:
This may have been left after a rebase when the `closure_reason` logic above was added. Anyhow it's a mistake now, thanks!